sympy/physics/units/__init__.py

```diff
@@ -28,12 +27,13 @@
     expression expressed in another unit.
 
 """
+# isort:skip_file
```

Comment:
It's related only to `isort` utility which helps with keeping the imports well organized. I can move it to the top if you prefer.